examples/set-bucket-lifecycle.mjs

```diff
@@ -67,9 +63,5 @@ const lifecycleConfigWithExpirationDate = {
   ],
 }
 
-s3Client.setBucketLifecycle('bucketname', lifecycleConfigWithExpirationDate, function (err) {
-  if (err) {
-    return console.log(err)
-  }
-  console.log('Success')
-})
+const ilmDatePolicyRes = await s3Client.setBucketLifecycle('test-bucket', lifecycleConfigWithExpirationDate)
+console.log(ilmDatePolicyRes)
```

Comment:
In the same vein as my previous comment, the `console.log` statements in this example seem to be returning `undefined` for me. This is expected, right?